lib/java/deploy/MavenPerBranchSpringBootDeploymentGoal.ts

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-import { logger } from "@atomist/automation-client";
+import {HandlerContext, logger, Success} from "@atomist/automation-client";
```

Comment:
Typically we break multiple imports into multiple lines, one for each named import.

Reply:
Done